Source/WebCore/platform/graphics/avfoundation/objc/MediaPlayerPrivateMediaStreamAVFObjC.mm

```diff
@@ -1157,6 +1157,22 @@ void updateTracksOfKind(MemoryCompactRobinHoodHashMap<String, RefT>& trackMap, T
     return metadata;
 }
 
+LayerHostingContextID MediaPlayerPrivateMediaStreamAVFObjC::hostingContextID() const
+{
+    if (m_sampleBufferDisplayLayer)
+        return m_sampleBufferDisplayLayer->hostingContextID();
+    return { };
+}
+
+void MediaPlayerPrivateMediaStreamAVFObjC::setVideoInlineSizeFenced(const FloatSize& size, const WTF::MachSendRight&)
```

Comment:
I am not familiar with setVideoInlineSizeFenced. I see this is related to full screen.
Why aren't we using the second parameter like for the remote player code path, for instance to RemoteSampleBufferDisplayLayer::m_layerHostingContext?

Reply:
> I am not familiar with setVideoInlineSizeFenced. I see this is related to full screen. Why aren't we using the second parameter like for the remote player code path, for instance to RemoteSampleBufferDisplayLayer::m_layerHostingContext?

That is a good point. I have addressed this in the latest patch by sending the fence back to the GPU process.

Thanks for reviewing!